view/frontend/web/js/view/payment/method-renderer/razorpay-method.js

```diff
@@ -182,7 +182,9 @@ define(
                     "po_number": null,
                     "additional_data": {
                         rzp_payment_id: this.rzp_response.razorpay_payment_id,
-                        order_id: this.merchant_order_id
+                        order_id: this.merchant_order_id,
+                        rzp_order_id: this.rzp_response.razorpay_order_id,
```

Comment:
`rzp_order_id` is not used on the backend. It shouldn't be sent because it makes screwing up the signature very easy.